crawl-ref/source/art-data.txt

```diff
@@ -451,6 +456,9 @@ TILE_EQ:  glaive_of_the_guard
 BRAND:    SPWPN_ELECTROCUTION
 AC:       5
 INSCRIP:  spectral,
+DESCRIP:  It retains the spirit of the tree from which it was made. In the hands
+ of one skilled in evocations this spirit is drawn out to fight along side the
+ wielder.
```

Comment:
The description of the spectral has changed, this should match the spectral brand description.

Reply:
I've changed this now.